.test_env_config_fact_store.yaml

```diff
@@ -0,0 +1,9 @@
+STORAGE_BACKEND: "local"
+LS_INPUT_PATH: "validation_data/log_anomaly_detector-100000-events.json"
+LS_OUTPUT_PATH: "validation_data/local-results-1.txt"
+INFER_ANOMALY_THRESHOLD: 1.3
+INFER_TIME_SPAN: 1
+INFER_LOOPS: 1
+PARALLELISM: 6
+FREQ_NOISE: 10000
```

Comment:
can we add the new settings, like `FREQ_NOISE` to the readme to explain what they are doing?

Reply:
The purpose is that we want to increase the occurrences of log lines that are false positives so the SOM model will give it a lower score thus not triggering an false anomaly prediction. We have an additional check in place. I guess we can probably measure how well this performs overtime. W2v and SOM ML Models should learn overtime  ![Anomaly_False_Positive](https://user-images.githubusercontent.com/1269759/58996555-b3b88700-87c6-11e9-8e0d-0bf947e02699.png)

Reply:
good. add that explanation to the readme, where the other variables are explained

Reply:
Done